smqtk_relevancy/__init__.py

```diff
@@ -0,0 +1,4 @@
+import pkg_resources
```

Comment:
Continuing with the theme, lets remove this import and the `__version__` attr below.